database/surrealdb/surrealdb.go

```diff
@@ -0,0 +1,289 @@
+package surrealdb
+
+import (
+	"fmt"
+	"io"
+	nurl "net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/hashicorp/go-multierror"
+	"github.com/surrealdb/surrealdb.go"
```

Comment:
Do you know if the surrealdb go client supports multiple platforms?
My guess is that it does since it seems to be implemented in pure Go.
You can test this by running `make build-cli` or more comprehensively with `DATABASE=$(make echo-database) SOURCE=$(make echo-source) goreleaser release --clean --snapshot --skip-docker`

Reply:
I believe that yes it supports multiple platforms here is the output of `DATABASE=$(make echo-database) SOURCE=$(make echo-source) goreleaser release --clean --snapshot --skip-docker`

```
  • starting release...
  • loading                                          path=.goreleaser.yml
  • DEPRECATED: --skip-docker was deprecated in favor of --skip=docker, check https://goreleaser.com/deprecations#-skip for more details
  • skipping announce, docker, publish and validate...
  • loading environment variables
  • getting and validating git state
    • ignoring errors because this is a snapshot     error=git doesn't contain any tags. Either add a tag or use --snapshot
    • git state                                      commit=bcdd2f120462bec5a83b31c7f8a611d44e963be2 branch=surrealdb current_tag=v0.0.0 previous_tag=<unknown> dirty=false
    • pipe skipped                                   reason=disabled during snapshot mode
  • parsing tag
  • setting defaults
  • snapshotting
    • building snapshot...                           version=v0.0.0-next
  • running before hooks
    • running                                        hook=go mod tidy
    • took: 4s
  • checking distribution directory
  • loading go mod information
  • build prerequisites
  • writing effective config file
    • writing                                        config=dist/config.yaml
  • building binaries
    • building                                       binary=dist/migrate_darwin_arm64/migrate
    • building                                       binary=dist/migrate_linux_amd64_v1/migrate
    • building                                       binary=dist/migrate_windows_arm_7/migrate.exe
    • building                                       binary=dist/migrate_linux_arm64/migrate
    • building                                       binary=dist/migrate_darwin_amd64_v1/migrate
    • building                                       binary=dist/migrate_windows_arm64/migrate.exe
    • building                                       binary=dist/migrate_windows_amd64_v1/migrate.exe
    • building                                       binary=dist/migrate_windows_386/migrate.exe
    • building                                       binary=dist/migrate_linux_386/migrate
    • building                                       binary=dist/migrate_linux_arm_7/migrate
    • took: 1m11s
  • archives
    • creating                                       archive=dist/migrate.linux-386.tar.gz
    • creating                                       archive=dist/migrate.linux-armv7.tar.gz
    • creating                                       archive=dist/migrate.windows-arm64.zip
    • creating                                       archive=dist/migrate.linux-arm64.tar.gz
    • creating                                       archive=dist/migrate.windows-amd64.zip
    • creating                                       archive=dist/migrate.windows-armv7.zip
    • creating                                       archive=dist/migrate.windows-386.zip
    • creating                                       archive=dist/migrate.darwin-arm64.tar.gz
    • creating                                       archive=dist/migrate.linux-amd64.tar.gz
    • creating                                       archive=dist/migrate.darwin-amd64.tar.gz
    • took: 6s
  • creating source archive
    • creating source archive                        file=migrate-v0.0.0-next.zip
  • linux packages
    • creating                                       package=migrate format=deb arch=arm64 file=dist/migrate.linux-arm64.deb
    • creating                                       package=migrate format=deb arch=386 file=dist/migrate.linux-386.deb
    • creating                                       package=migrate format=deb arch=arm7 file=dist/migrate.linux-armv7.deb
    • creating                                       package=migrate format=deb arch=amd64v1 file=dist/migrate.linux-amd64.deb
    • took: 1s
  • calculating checksums
  • storing release metadata
    • writing                                        file=dist/artifacts.json
    • writing                                        file=dist/metadata.json
  • you are using deprecated options, check the output above for details
  • release succeeded after 1m21s
  • thanks for using goreleaser!
```